### PR TITLE
Add Sails.js

### PR DESCRIPTION
**Pull request by DaAwesomeP**
I categorized Koa and Express as web servers as well.

### CODE REVIEW

**Comment by DaAwesomeP** [comment]
The build errored. That's odd. Seems to have been unable to download something.

File: src/apps.json [hunk 1]
```diff
@@ -5663,6 +5665,21 @@
       "website": "www.sbuilder.ru",
       "icon": "S.Builder.png"
     },
+    "Sails.js": {
+      "cats": [
+        18
+      ],
+      "headers": {
+        "Set-Cookie": "^sails.sid$",
```
Comment (gadcam) [inline src/apps.json]: If you want to match a dot I think you'd better escape it.

Comment (DaAwesomeP) [inline src/apps.json]: My bad. I'll fix it.

File: src/apps.json [hunk 2]
```diff
@@ -5663,6 +5665,21 @@
       "website": "www.sbuilder.ru",
       "icon": "S.Builder.png"
     },
+    "Sails.js": {
+      "cats": [
+        18
+      ],
+      "headers": {
+        "Set-Cookie": "^sails\\.sid$",
```
Comment (DaAwesomeP) [inline src/apps.json]: @gadcam Fixed

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**